packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -219,7 +232,7 @@ const helpers = {
       findResponseByStatusCode(httpResponses, '400') ||
       createResponseFromDefault(httpResponses, '422');
     if (!response) {
-      throw new Error('No 422 response defined');
+      throw new Error('Nor 422, 400 or default response defined');
```

Comment:
"Neither Frank, nor Bob, knew how OpenAPI examples work."
```suggestion
      throw new Error('No 422, 400, or default responses defined');
```